core/Symbols.cc

```diff
@@ -812,6 +812,22 @@ SymbolRef Symbol::singletonClass(GlobalState &gs) {
     singletonInfo->setSuperClass(Symbols::todo());
     singletonInfo->setIsModule(false);
 
+    auto tp = gs.enterTypeMember(this->loc(), singleton, Names::Constants::AttachedClass(), Variance::CoVariant);
+
+    // Initialize the bounds of AttachedClass as top and bottom, as the upper
+    // bound will be fixed in the resolver's global pass.
```

Comment:
Do I read this `fixed` right as `mutated` not as `type_member(fixed: ...)`?

Reply:
You're correct, `fixed` as in `mutated`. I'll update the comment to avoid ambiguous language :)